decls/i18n.js

```diff
@@ -110,5 +110,5 @@ declare interface I18n {
 };
 
 declare interface Formatter {
-  interpolate (message: string, values?: any): Array<any>
+  interpolate (message: string, values: any, path: string): Array<any> | void
```

Comment:
In the case that interpolate return the nothing, I hope that it's better to be `null` than `undefined`.